src/test/java/org/tarantool/ClientReconnectIT.java

```diff
@@ -213,11 +214,54 @@ public void run() {
         });
     }
 
+    // DO NOT REMOVE THIS TEST
+    // Motivation: this test checks start/stop correctness
+    // of TarantoolControl class which is used by other tests.
+    // This test is commented out because the class is used
+    // for internal purposes only and isn't related to
+    // the connector testing.
+    //    @Test
+    //    @DisplayName("follow up the issue #164")
+    //    void testStartStopTarantoolInstance() throws InterruptedException {
+    //        int numberOfParallelInstances = 4;
```

Comment:
This test does not fail for me even with 50 threads. It seems that it is hard to step into the issue on some hardware / environment. But I would leave the test here anyway, because we will still able to check TarantoolControl if we'll suspect some issues with it in the future. Just we'll use Travis-CI machines or some hardware where we able to catch the issue.